app.js

```diff
@@ -52,3 +52,21 @@ capitalHill.makeRow();
 var alki = new Store('Alki', 2, 16, 4.6);
 alki.avgCookieHour();
 alki.makeRow();
+
+var formEl = document.getElementById('form')
+
+formEl.addEventListener('submit', handleSubmit);
+
+function handleSubmit(event){
+  event.preventDefault();
+  console.log(event.target.storeName.value);
+
+  var storeName = event.target.storeName.value;
+  var minCust = event.target.minCust.value;
```

Comment:
Something to keep in mind is that the values that come in from the input via the `.value` are going to be in string format which can mess up your calculations. For instance `'8' + 1 === '81'` not `9`.